### PR TITLE
Update tsne-plot package

### LINKED ISSUE #361: Showing gray color plot expression when we click on specific experiment for the first time page load
In the SCXA a few experiments don't have metadata. Those experiments have given empty colour plot expressions when clicking on the experiment.

`Ex: https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-GEOD-99058/results/tsne`

Here is the screenshot:

Image

Change the K value to something and return to the same k value that is `23`, then you will see a colour plot.

-  We are getting an empty colour plot for the metadata available experiments too.

Image

Empty colour plot  screen shot:

Image

### LINKED ISSUE #363: "Colour plot by" drop down corresponding endpoints(cluster and metadata) inconsistently returning 400 error when drop down options switched from  metadata to k value and vice versa
The result page gives 400 errors for the experiment 'E-HCAD-30'

These two endpoints behaving a weirdly:

```
/gxa/sc/json/cell-plots/E-HCAD-30/clusters/metadata/20?plotMethod=tsne&accessKey=&perplexity=50 => 400

/gxa/sc/json/cell-plots/E-HCAD-30/expression/?plotMethod=tsne&accessKey=&perplexity=50 => 400
```

Need to investigate to fix it.

Here is the screenshot:

image

### PR DESCRIPTION
**Pull request by lingyun1010**
The update in the tsne plot is here https://github.com/ebi-gene-expression-group/atlas-components/pull/144.

### CODE REVIEW

File: app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ class TSnePlotViewRoute extends React.Component {
       selectedColourBy: cellTypeValue ? cellTypeValue.toLowerCase() : this.props.ks[Math.round((this.props.ks.length -1) / 2)].toString(),
       highlightClusters: [],
       experimentAccession: this.props.experimentAccession,
-      selectedColourByCategory: !isNaN(search.colourBy) ? `clusters` : `metadata`,
-      selectedClusterByCategory: !isNaN(search.colourBy) ? `clusters` : `metadata`,
+      selectedColourByCategory: isNaN(search.colourBy) ? `metadata` :  `clusters`,
+      //selectedClusterByCategory: !isNaN(search.colourBy) ? `clusters` : `metadata`,
```
Comment (ke4) [inline app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js]: Could you please remove this line if we don't need it. Thanks

**Review by ke4 — CHANGES_REQUESTED** [review]
Just a tiny change request, other than that it is all fine. Thanks

**Review by upendrakumbham — DISMISSED** [review]
I tested this PR in my local after running front-end recompile and build and deploy scripts.
All looks great to me, actually Gray colour plot issue also fixed it seems. I didn't see it in the PR.

**Review by ke4 — APPROVED** [review]
LGTM

**Pull request merged**